src/output.ts

```diff
@@ -131,8 +145,22 @@ export class Output {
         return { ...test, test_code: null }
       }
 
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      return { ...test, test_code: testCase.testCode(parsedSource.source) }
+      // Version 3 optionally can output the task ID
+      if (this.configFlag('flag.tests.task-per-describe')) {
+        return {
+          ...test,
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          task_id: testCase.topLevelIndex,
```

Comment:
You already have this property available. :exploding_head: That makes things very easy.

Reply:
Haha, I added it last night!